api/agent/protocol/json.go

```diff
@@ -33,6 +33,8 @@ func writeString(err error, dst io.Writer, str string) error {
 	return err
 }
 
+// TODO(xxx): headers, query parameters, body - what else should we add to func's payload?
```

Comment:
Request url request method shouldnt they be included?

Reply:
They are part of env: FN_METHOD, FN_REQUEST_URL, etc.

Reply:
K

Reply:
Just made sense to have at top level to me but it's the either way so :).

Reply:
Other thing is auth: user/password from URL structure

Reply:
Request url should be in Json since it can change every request.

Reply:
@treeder most parts of URL already in env like HTTP method, request URL, which parts of it should also be included?